pyccel/ast/core.py

```diff
@@ -2149,17 +2156,17 @@ def rename(self, newname):
 
         self._name = newname
 
-    def add_local_var(self, var):
+    def add_local_var(self, *var):
         """
-        Add a new variable to the local variables tuple
+        Add (a) new variable(s) to the local variables tuple
 
         Parameters
         ----------
         var : Variable
               The new local variable
         """
-        var.set_current_user_node(self)
-        self._local_vars = self._local_vars + (var,)
+        _ = [v.set_current_user_node(self) for v in var]
+        self._local_vars = self._local_vars + var
 
```

Comment:
I suggest we change the name of this function to `add_local_vars` (plural), which is consistent with other function names in the parser and printers.

Reply:
I would also call the argument `vars` (plural) for the same reason.

Reply:
`vars` is a [builtin function](https://docs.python.org/3/library/functions.html#vars) in python

Reply:
Fixed in d872f4f